### PR TITLE
Fixing add profile repeater not enabled for fetched speaker profile

### PR DESCRIPTION
**Pull request by gbajaj91**
isPlaceholder is returned as true for some saved profiles. ideally it should not be stored and returned from backend. Adding checks to prevent handling of this.
And other minor fixes
1. Sending social links instead of social media in speaker body
2. Showing error toast when speaker update/create fails
3. Showing error when image is > 25MB
4. Showing error when social media is out of the supported list

Resolves: [MWPW-154773](https://jira.corp.adobe.com/browse/MWPW-154773)

Test URLs:
- Before: https://main--ecc-milo--adobecom.hlx.page/
- After: https://gbajaj-bugfix--ecc-milo--adobecom.hlx.page/

### CODE REVIEW

File: ecc/scripts/esp-controller.js [hunk 1]
```diff
@@ -275,14 +296,13 @@ export async function removeSpeakerFromEvent(speakerId, eventId) {
...
+  const nSpeaker = convertToNSpeaker(profile);
   const { host } = getAPIConfig().esp[ECC_ENV];
-  const nProfile = { ...profile, photo: undefined };
-  const raw = JSON.stringify({ ...nProfile, seriesId });
+  const raw = JSON.stringify({ ...nSpeaker, seriesId });
   const options = await constructRequestOptions('PUT', raw);
 
   const resp = await fetch(`${host}/v1/series/${seriesId}/speakers/${profile.speakerId}`, options)
-    .then((res) => res.json())
-    .catch((error) => window.lana?.log(`Failed to update speaker. Error: ${error}`));
```
Comment (qiyundai) [inline ecc/scripts/esp-controller.js:285]: Any reason we removed the lana log here?

**Review by qiyundai — COMMENTED** [review]
Just have one quick question on a lana line removal. Other than that LGTM.

**Comment by qiyundai** [comment]
> Just have one quick question on a lana line removal. Other than that LGTM.

And we can probably remove the comment lines

File: ecc/scripts/esp-controller.js (hunk 1 above)
Comment (gbajaj91) [reply ecc/scripts/esp-controller.js]: unintentional, thanks for catching this.

**Review by qiyundai — APPROVED** [review]

**Pull request merged**